quickstart: remove optionals settings for certificates, and cookie secrets (#161)
pomerium/documentation: content/docs/install/quickstart.mdx

Docker will automatically download the required [container images] for Pomerium and [verify]. Then, Pomerium will run with the configuration details set in the previous steps.

You should now be able to access the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.

> ### Handle HSTS Errors

Reviewer: ```suggestion
### Handle Self-Signed Certificate Warning
```

> When navigating to the `https://verify.localhost.pomerium.io` route defined in your policy, you may encouter the following HSTS error:

Reviewer: ```suggestion
When navigating to the `https://verify.localhost.pomerium.io` route defined in your policy, you may encounter the following self-signed certificate warning:
```

> ![hsts error](./img/hsts-error.png)

Reviewer: ```suggestion
![self-signed warning](./img/hsts-error.png)
```

To resolve this error:
1. Click anywhere inside the browser window
2. Type **thisisunsafe**
3. Make sure **Reload** is selected
3. Press **ENTER**

Your browser will redirect you to the route defined in your policy: 

> ![user details page](./img/user-details-page.png)

Author: I installed Shottr, WAY cleaner. I'm updating the image.
